app/assets/javascripts/components/option-select.js

```diff
@@ -1,14 +1,15 @@
-(function ($) {
-  "use strict";
-  window.GOVUK = window.GOVUK || {};
-
-  function OptionSelect(options){
-    /* This JavaScript provides two functional enhancements to option-select components:
-      1) A count that shows how many results have been checked in the option-container
-      2) Open/closing of the list of checkboxes - this is not provided for ie6 and 7 as the performance is too janky.
-    */
-
-    this.$optionSelect = options.$el;
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {}; // if this ; is omitted, none of this JS runs :(
```

Comment:
We might not have a `;` because, as far as I can tell, we're supposed to use [Standard for linting](https://github.com/alphagov/styleguides/blob/master/js.md#linting), which doesn't allow semicolons.

Reply:
Not saying that's the case here - but it's a helpful comment to have in case a future-person comes along and reformats this to make it Standard / other linter compliant.

Reply:
Interesting - thanks for the explanation!

Reply:
I've added in all the missing semicolons since the rest of the file has them, but yeah, at some point we're going to switch everything to using standard. @alex-ju [opened a PR](https://github.com/alphagov/govuk_publishing_components/pull/758) a while ago to do this for the components gem, but I'm not sure what's happening with it.